lib/toFixed.js

```diff
@@ -12,5 +12,9 @@
  */
 
 module.exports = function toFixed (num) {
-  return num.toFixed(2);
+  var int = parseFloat(num);
```

Comment:
If we're going to use `parseFloat` (in which case we will accept strings), we should update the associated comment to denote that the parameter may be `{Number|String}` now.